### PR TITLE
Rm 88

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
shared component ClientSalesTransaction read only table , test : sale invoince , enable when there is formik.values.header.clientId

### CODE REVIEW

File: src/components/Shared/FormShell.js [hunk 1]
```diff
@@ -279,6 +280,19 @@ export default function FormShell({
               title: platformLabels.ClientBalance
             })
           }
+          onClientSalesTransaction={() =>
```
Comment (imtithalhalalo) [inline src/components/Shared/FormShell.js:283]: when the RM-75 is merged you need to change this one to be similar to how we are doing the functions in FormShell

File: src/components/Shared/ClientSalesTransaction.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      extension: SaleRepository.SATrx.qry2,
+      parameters: `_functionId=${functionId}&_recordId=${recordId || 0}&_clientId=${clientId}`
+    })
+  }
+
+  const {
+    query: { data },
+    labels: _labels,
+    access
+  } = useResourceQuery({
```
Comment (imtithalhalalo) [inline src/components/Shared/ClientSalesTransaction.js:24]: you are missing endpoint

File: src/components/Shared/FormShell.js (hunk 1 above)
Comment (Alinadimmokdad) [reply src/components/Shared/FormShell.js:283]: okay

**Comment by imtithalhalalo** [comment]
Screenshot 2024-11-20 125640
once I open the plus button or edit

**Review by imtithalhalalo — APPROVED** [review]

**Pull request merged**